src/bill.js

```diff
@@ -0,0 +1,90 @@
+//package
+const prompt = require('prompt-sync')();
+/**
+ * class to generate a bill object with the following structure: 
+ * Bill{name: @{Number}, type: @{String}, numbers: @{String}, city: @{String}}
+ * type: @{String} > one of 'ambata','ambo','terno','quaterna','cinquina'
+ * numbers: n random numbers, n is defined by the user
+ * city: one of > 'Bari','Cagliari','Firenze','Genova','Milano','Napoli','Palermo','Roma','Torino','Venezia'
+ */
+class Bill {
+    constructor(billNumber,city,type,numbers){
+        this.name = billNumber
+        this.city = city
+        this.type = type
+        this.numbers = numbers
+    }
+    static selectCity(){
+        const cities = ['Bari','Cagliari','Firenze','Genova','Milano','Napoli','Palermo','Roma','Torino','Venezia']
+        let city = prompt(`Please enter a city for the bill: `,'')
+        while (cities.includes(city)!=true){
+            city = prompt(`Please enter a VALID city for the bill from the following list: ${cities.join(', ')}.`,'')
+        }
+        return city
+    }
+    static selectType(){
+        const typeOfBills = ['ambata','ambo','terno','quaterna','cinquina']
+        let type = prompt(`Please enter a type for the bill: `,'')
+        while (typeOfBills.includes(type)!=true){
+            type = prompt(`Please enter a VALID type for the bill from the following list: ${typeOfBills.join(', ')}. `,'')
+        }
+        return type
+    }
+    static selectQuantity(){
+        const minNum = 1
+        const maxNum = 10
+        let quantity = +parseInt(prompt('Please enter the number of numbers you want to play between 1 and 10: ',0))
+        while((quantity<minNum || quantity>maxNum) || isNaN(quantity)){
+            quantity = +parseInt(prompt('Please enter a number between 1 and 10: ',0))
+        }
+        return quantity
+    }
+    static selectNumbers(){
+        const maxNum = 90
+        let numExtraction = this.selectQuantity()
+        const numbersArry = new Set() //unique values
+
+        while (numExtraction!=0){
+            const randomNum = Math.floor(Math.random()*maxNum)+1
+            numbersArry.add(randomNum)
+            numExtraction--
+        }
+        return [...numbersArry].join(',')
+    }
+    /**
+     * method to print bills
+     * @param {Array of Object} bills 
+     * @returns {String} console.log the final result in a nice template
+     */
+    static printBill(bills){
```

Comment:
This method also doesn't have to be static. 
It should **print a single ticket**.
And you will call it with the dot notation (` bill.printBill() `) on every single ticket/bill.